closure/goog/math/box.js

```diff
@@ -174,6 +170,17 @@ goog.math.Box.prototype.expandToInclude = function(box) {
   this.bottom = Math.max(this.bottom, box.bottom);
 };
 
+/**
+ * Expand this box to include the coordinate.
+ * @param {!goog.math.Coordinate} coord The coordinate to be included
+ * inside the box.
```

Comment:
indent 4. See examples throughout file.

In the future, you should try running the [closure linter](https://github.com/google/closure-linter) on your PR files. Will speed things up :). Eventually, this will be an automated step in Travis for all PRs, but for now you can run manually.